### PR TITLE
loops_homework

### PR DESCRIPTION
**Pull request by OleksandrBochenko**

### CODE REVIEW

File: loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+my_list = [1, 2, 3, 4, 6, 7, 8]
+filtered = []
+for element in range(0, len(my_list) - 1):
+    if my_list[element] + 1 != my_list[element + 1]:
+         print(my_list[element + 1])
```
Comment (MarinaZelenska) [inline loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py]: You should use formatting string

File: loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+         break
```
Comment (MarinaZelenska) [inline loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py]: What if list is consecutive? You should add block else for loop
image

File: loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+my_list = [1, 2, 3, 4, 6, 7, 8]
+filtered = []
```
Comment (MarinaZelenska) [inline loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py]: We don't use this filtered list. Delete it

**Review by MarinaZelenska — APPROVED** [review]

File: loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py (hunk 1 above)
Comment (OleksandrBochenko) [reply loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py]: fixed

File: loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py (hunk 2 above)
Comment (OleksandrBochenko) [reply loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py]: fixed

File: loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py (hunk 3 above)
Comment (OleksandrBochenko) [reply loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py]: fixed

**Pull request merged**